localization/src/languages/en.json

```diff
@@ -400,6 +400,7 @@
     "Submit": "Submit",
     "Subscribers": "Subscribers",
     "Success": "Success",
+    "TnftNotSupportedMedia": "Surf will soon learn to display this TNFT",
```

Comment:
To be honest I don't like that UIKit knows anything about `Surf`.

I'd make an error message more generic here and let the component to accept an error message from the outside (i.e. Surf), and display this message only in case it wasn't provided via props.

@savelichalex what do you think?

Reply:
Agree, 100%